CONTRIBUTING.adoc

```diff
@@ -117,6 +117,7 @@ To facilitate rapid development we've put together a Vagrantfile you can use to
 
 3.  Clone the project and change into the directory:
 
+        # assumes something like `export GOPATH=$HOME/go` is set in your .bash_profile
```

Comment:
This is set for you in our Vagrant environment, so we probably don't need the comment.